### PR TITLE
DFR-2457 adding backwards compatibility for hearing court

### PR DESCRIPTION
**Pull request by gmikerlemos**
https://tools.hmcts.net/jira/browse/DFR-2457

### CODE REVIEW

File: definitions/contested/json/CaseTypeTab/CaseTypeTab.json [hunk 1]
```diff
@@ -3849,7 +3849,7 @@
     "TabLabel" : "Scheduling and Listing",
     "TabDisplayOrder" : 15,
     "CaseFieldID" : "chooseCourtLabelSL",
-    "FieldShowCondition" : "hearing_regionList!=\" \"",
```
Comment (rishikrsharma) [inline definitions/contested/json/CaseTypeTab/CaseTypeTab.json:3852]: These change will only be visible to caseworker only. There are 6 tabs based on roles. Please check i think It has to be visible to all eligible parties.

Comment (gmikerlemos) [reply definitions/contested/json/CaseTypeTab/CaseTypeTab.json:3852]: sorted on latest commit

File: definitions/contested/json/CaseTypeTab/CaseTypeTab.json [hunk 2]
```diff
@@ -4393,7 +4674,7 @@
     "TabDisplayOrder": 15,
     "CaseFieldID": "nottinghamCourtList",
     "TabFieldDisplayOrder": 15,
-    "FieldShowCondition": "regionList=\"midlands\" AND midlandsFRCList=\"nottingham\""
+    "FieldShowCondition": "hearingDate !=\"\" AND hearing_regionList=\"\" AND regionList=\"midlands\" AND midlandsFRCList=\"nottingham\""
```
Comment (rishikrsharma) [inline definitions/contested/json/CaseTypeTab/CaseTypeTab.json:4677]: Is this condition correct? 
If regionList =="" and hearing_regionList="midlands" for new cases this condition will fail.

similarly others.

Comment (gmikerlemos) [reply definitions/contested/json/CaseTypeTab/CaseTypeTab.json:4677]: I think so, this field is not supposed to show on new cases

**Pull request merged**